students/shirin_ak/Session06/test_mailroom.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from mailroom import create_report,send_thank_you_letter,add_donor
+
+
+
+def test_send_thank_you_letter():    
+    expected = "David,Thank you for your most recent donation of $2500.17."
+    assert send_thank_you_letter("David", 2500.17)== expected 
+
+
+
+
+def test_display_report():
+    report = create_report() 
+    assert create_report()== report
```

Comment:
For this one, _report_ should be assigned the expected string that _create_report()_ will output, the way it is currently code, there's no way it would fail, as _report_ and _create_report()_ reference each other.